engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/transformation/BpmnTransformer.java

```diff
@@ -98,6 +98,7 @@ public BpmnTransformer(final ExpressionLanguage expressionLanguage) {
     step3Visitor.registerHandler(new SubProcessTransformer());
 
     step4Visitor = new TransformationVisitor();
+    step4Visitor.registerHandler(new ContextProcessTransformer());
```

Comment:
@remcowesterhoud Did the conflict occur here?

Reply:
Yes. In previous stable versions we didn't have the `IntermediateThrowEventTransformer` in this step. Since we do have this on main atm it conflicted.